test/unit/SelectedFilters.test.js

```diff
@@ -15,6 +15,40 @@ describe('SelectedFilters', () => {
     });
   });
 
+  describe('No JS - clear all link', () => {
+    const selectedFilters = {
+      language: [
+        {
+          value: 'lang:en',
+          label: 'English',
+          count: 4267,
+        },
+      ],
+      materialType: [],
+      dateBefore: '',
+      dateAfter: '',
+    };
+    let component;
+
+    before(() => {
+      component = shallow(<SelectedFilters selectedFilters={selectedFilters} />);
+    });
+
+    it('should render a ul', () => {
+      expect(component.find('ul').length).to.equal(1);
+    });
+
+    it('should render three list items, one text, one with data, one clear link', () => {
+      expect(component.find('li').length).to.equal(3);
+    });
+
+    it('should have a clear all filters link', () => {
+      expect(listItemAt(component, 2).find('a').length).to.equal(1);
+      expect(listItemAt(component, 2).find('a').render().text())
+        .to.equal('Clear FiltersRemove Filter');
```

Comment:
It passes the tests, but should the text be `Clear FiltersRemove Filter`?

Reply:
Right, that's a bit odd, but `Remove Filter` the label for the X icon...
I'm updating it and hiding that from screenreaders, but for the unit tests it's going to render all of it.